api/Element.json

```diff
@@ -8137,7 +8138,8 @@
             ],
             "firefox_android": [
               {
-                "version_added": false
+                "version_added": true,
```

Comment:
@ddbeck @chrisdavidmills  I've had [an answer](https://bugzilla.mozilla.org/show_bug.cgi?id=1662124#c10)  from engineer about pointer capture support on FF for Android:

> The story for FF Android is a bit different, we didn't ship pointer event on FF Android along with FF Desktop from 59.
We shipped it while we move FF Android to GeckoView, see https://bugzilla.mozilla.org/show_bug.cgi?id=1507495. But since it depend on runtime status on FF Android, so we need to update the pref dynamically.

I tested the latest version of FF for Android on Google Play and it pointer events works (which is why it is added as true) here.
- Do you understand what the engineer means by " we need to update the pref dynamically."?
- In terms of the version it went in as supported we know that it wasn't 59. bug 1507495](https://bugzilla.mozilla.org/show_bug.cgi?id=1507495), indicates it comes in with Gecko in Gecko 70. Do you know how I can find out the mapping between gecko versions and firefox for Android versions. Otherwise I guess we could merge as true?

Reply:
@ddbeck If not, then I propose we accept this as is with "true". 

Though *personally*. I'd find it more useful to have the current version than just true, since going forward people would know that it worked in version 82, while if you put in "true" then all they know is that it works in the current version.

Reply:
@hamishwillee Thank you for your patience while I came back to this. Some answers, finally!

> Do you understand what the engineer means by " we need to update the pref dynamically."?

No, that's mysterious to me. That said, the issue you mentioned after this seems to settle the question of "when did this ship to all users?"

> …[bug 1507495](https://bugzilla.mozilla.org/show_bug.cgi?id=1507495), indicates it comes in with Gecko in Gecko 70. Do you know how I can find out the mapping between gecko versions and firefox for Android versions.

We actually include this in BCD's browser data! For example, [`firefox_android.json`](https://github.com/mdn/browser-compat-data/blob/master/browsers/firefox_android.json). This is very helpful for syncing up, for example, Opera desktop and Android, which have some slippage between them. Though, helpfully, Gecko versions are equivalent to Firefox versions from version 5 onward.

> Though personally. I'd find it more useful to have the current version than just true, since going forward people would know that it worked in version 82, while if you put in "true" then all they know is that it works in the current version.

Yeah, I get where you're coming from. In terms of BCD's schema, however, `true` always means "unsupported, but initial version unknown". `true` (and `null`) versions are incomplete, but recognizably so (we even track this sort of thing, see https://github.com/mdn/browser-compat-data/issues/3555#issuecomment-698485184 for an example); subbing in the current version for `true` is silently wrong, which I actively avoid.

But that's academic in this case, since it sounds like 70 is a safe value to set here. 🏁